src/dtproject/types.py

```diff
@@ -119,14 +120,15 @@ class LayerRecipes(DictLayer[Recipe]):
     ITEM_CLASS = Recipe
 
 
+@dataclasses.dataclass
 class ContainerConfiguration(dict):
-    pass
+    extends: Optional[str]
+    plain: Optional[bool]
```

Comment:
These fields are not required in the schema (which is correct), but they don't have a default value here. Parsing a file where one of those fields is missing will fail. Add default values `None` to both fields.